general/device.hpp

```diff
@@ -123,20 +123,27 @@ class Device
 
    static bool device_env, mem_host_env, mem_device_env;
    static Device device_singleton;
+#ifdef MFEM_USE_UMPIRE
+   static bool use_host_umpire;
+   static bool use_device_umpire;
+#endif
 
-   MODES mode;
+   MODES mode{Device::SEQUENTIAL};
```

Comment:
Can we not mix initialization styles?

Reply:
Can you clarify: do you mean use either ` = val` or `{val}` but not both? Is one preferred over the other?

Reply:
Yes the current style is `= val`.

Reply:
Updated